tank/tflite/albert.py

```diff
@@ -0,0 +1,33 @@
+# RUN: %PYTHON %s
+import numpy as np
+from shark.shark_importer import SharkImporter
+import pytest
+
+model_path = "https://tfhub.dev/tensorflow/lite-model/albert_lite_base/squadv1/1?lite-format=tflite"
+
+# Inputs modified to be useful albert inputs.
+def generate_inputs(input_details):
+    for input in input_details:
+        print("\t%s, %s", str(input["shape"]), input["dtype"].__name__)
+
+    args = []
+    args.append(
+        np.random.randint(low=0, high=256, size=input_details[0]["shape"], dtype=input_details[0]["dtype"]))
+    args.append(np.ones(shape=input_details[1]["shape"], dtype=input_details[1]["dtype"]))
+    args.append(np.zeros(shape=input_details[2]["shape"], dtype=input_details[2]["dtype"]))
+    return args
+
+if __name__ == '__main__':
+    my_shark_importer = SharkImporter(model_path=model_path,
+                                      model_type="tflite",
+                                      model_source_hub="tfhub",
+                                      device="cpu",
+                                      dynamic=False,
+                                      jit_trace=True
+                                      )
+    input_details, output_details = my_shark_importer.get_model_details()
+    inputs = generate_inputs(input_details) # device_inputs
+    my_shark_importer.setup_inputs(inputs)
```

Comment:
Can we put setup_inputs into class intialization of sharkImport S.T we only use this dummy_input for the shape information.

Reply:
We can add default inputs. but the generate_inputs call should be here, in case someone what other inputs

Reply:
yeap generate_inputs outside, setup_inputs inside I was thinking :)

Reply:
for example in SharkRunner, we set the dummy_input basically before compiling here in the init
https://github.com/nod-ai/SHARK/blob/main/shark/shark_runner.py#L46